cmds/core/wget/wget.go

```diff
@@ -45,16 +45,16 @@ func usage() {
 	os.Exit(2)
 }
 
-func run() (reterr error) {
+func run(args []string) (reterr error) {
 	log.SetPrefix("wget: ")
 
-	if flag.Parse(); flag.NArg() != 1 {
+	if len(args) != 1 {
 		usage()
 	}
 
-	argURL := flag.Arg(0)
+	argURL := args[0]
```

Comment:
does this work if args is nil?

Reply:
or if len(args) is 0?

Reply:
Yes, args can't be nil, we check it length on line 51, so it's only check if someone send `wget ""`. It's probably can be improved in url parsing.

I will try to address it in next PR, at least wget has a bug
```sh
./wget localhost:8080/200
wget: 2022/10/28 07:39:37 failed to download localhost:8080/200: encountered error no such scheme with "localhost:8080/200"
```